src/ralph/data_importer/resources.py

```diff
@@ -233,7 +234,7 @@ class DataCenterAssetResource(RalphModelResource):
     rack = fields.Field(
         column_name='rack',
         attribute='rack',
-        widget=ImportedForeignKeyWidget(physical.Rack),
+        widget=RackWidget(physical.Rack),
```

Comment:
No, it prints `[08:52:53] WARNING Record with pk 1 not found for model assetmodel of 'model'` and `Some of records were not imported`. It fails too without that flag.